komga-webui/src/locales/en.json

```diff
@@ -455,7 +455,7 @@
     },
     "server_stop": {
       "button_confirm": "Stop",
-      "confirmation_message": "Are you sure you want to stop Komga?",
```

Comment:
"Are you sure" has fallen out of fashion. Shorter.

Reply:
> "Are you sure" has fallen out of fashion

Where?

Reply:
Any design-guideline or modern program.
It invokes nagging and removes confidence.

Reply:
Source?

Reply:
https://material.io/components/dialogs#anatomy and so on.
Try not distrusting advice that is in no way harmful.